webapp/locales/de.json

```diff
@@ -515,7 +515,12 @@
     "search": {
         "placeholder": "Suchen",
         "hint": "Wonach suchst Du?",
-        "failed": "Nichts gefunden"
+        "failed": "Nichts gefunden",
+        "heading": {
+          "Post": "Beiträge",
+          "User": "Benutzer",
+	  "Tag": "Hashtags"
```

Comment:
```suggestion
	    "Tag": "Hashtags"
```

Reply:
the indentation is incorrect here, and we don't need this in this `PR`, aka code creep